test/evergreen.yml

```diff
@@ -255,8 +257,8 @@ functions:
           # This is useful as not all branches have a dedicated Evergreen project. Furthermore, the
           # documentation-update task is not triggered by every commit, we rely on the activity of
```

Comment:
commit, we --> commit. We

Reply:
Thanks, corrected in ed593ccc050d835888324ecda4396e4f18c41316